docs/tutorial/csrman_design.md

```diff
@@ -1,21 +1,119 @@
+
+|  signal name         |  channel type  | description                                  |
+| :------------------: | :------------: | :------------------------------------------: |
+| csr_req_addr_i[31:0] | req            |  Register address                            |
+| csr_req_data_i[31:0] | req            |  Data to write unto the specified address    |
+| csr_req_write_i      | req            |  Write/read signal: 1 - write, 0 read        |
+| csr_req_valid_i      | req            |  Transaction is valid                        |
+| csr_req_ready_o      | req            |  CSR manager is ready to accept transactions |
+| csr_rsp_data_o[31:0] | rsp            |  Data in response to reads                   |
+| csr_rsp_valid_o      | rsp            |  Response data is valid                      |
+| csr_rsp_ready_i      | rsp            |  Core side is ready to accept transactions   |
+
+
+The second side is the **(4) accelerator datapath interface**. The CSR manager sends the configurations of the read-write (RW) registers through a decoupled interface. However, the accelerator may have read-only (RO) registers that are meant for status monitoring. These are directly wired from the accelerator towards the CSR manager but without the need for a decoupled interface. When a user reads one of the RO registers, the CSR manager directly transmits the wired RO signals to the response ports. The table below describes these signals:
+
+|  signal name                      | description                                         |
+| :-------------------------------: | :-------------------------------------------------: |
+| csr_reg_rw_set_o [NumRwCsr][31:0] |  Packed RW configurations to send to accelerator    |
+| csr_reg_set_valid_o               |  Configurations are valid                           |
+| csr_reg_set_ready_i               |  Accelerator side is ready to accept configurations |
+| csr_reg_ro_set_i [NumRoCsr][31:0] |  Packed RO register data                            |
+
+The `NumRwCsr` and `NumRoCsr` pertain to the number of RW and RO registers, respectively.
+
+# CSR Manager Features
+
+The CSR manager has a set of read-write (RW) registers and read-only (RO) registers. Our CSR manager has a mechanism that bulk transfers a set of configurations to the main accelerator. The last RW register is always the start or send signal to transact all configurations into the accelerator. The figure below demonstrates this:
+
+![image](https://github.com/KULeuven-MICAS/snitch_cluster/assets/26665295/b9c4758a-ad20-4e3b-86f8-5dc7b649df09)
+
+In our [SNAX ALU accelerator](./accelerator_design.md), we have 2 main configuration RW registers: the mode and the data length to process. The 3rd RW register is the start that sends the configured settings to the accelerator data path. It is only when you assert the LSB of the last RW register (or the start register) that the CSR manager transacts all configurations to the accelerator.
```

Comment:
`assert the LSB of the last RW register (or the start register)`, I think `or` here might confuse there are two registers possible to start. It is actually, `aka`, start register.